packages/create-nuxt3-app/README.md

```diff
@@ -1,4 +1,61 @@
 # create-nuxt3-app
 
-See more information about usage on the main [ReadMe](../../README.md).
+[![npm version][npm-version-src]][npm-version-href]
+[![npm downloads][npm-downloads-src]][npm-downloads-href]
+[![GitHub stars](https://badgen.net/github/stars/sidebase/create-nuxt3-app)](https://GitHub.com/sidebase/create-nuxt3-app/)
+[![License][license-src]][license-href]
+[![Follow us on Twitter](https://badgen.net/badge/icon/twitter?icon=twitter&label)](https://twitter.com/sidebase_io)
+[![Join our Discord](https://badgen.net/badge/icon/discord?icon=discord&label)](https://discord.gg/9MUHR8WT9B)
 
+> Quickly bootstrap a new Nuxt3 project with the power of [sidebase](https://github.com/sidebase/sidebase)! Configure your custom tech stack and begin coding in minutes.
+
+## Usage
+
+```shell
+$ npx create-nuxt3-app
+```
+
+## Features
+
+- CI/CD Pipeline
+  - None
+  - GitHub Actions
+  - Drone (coming soon!)
+- Component Library
+  - None
+  - AntDesign Vue (currently included)
+  - PrimeVue (coming soon!)
+- Component Stories
+  - historie (currently included)
+  - Storybook (coming soon!)
+- Linting tools
+- Testing framework
+  - ViTest (currently included)
+
+We are always looking for more options to include! Join in on the conversation [here](https://github.com/sidebase/sidebase/issues/70)!
```

Comment:
maybe we can move that issue here / open a new one here and then reference the old one?

Reply:
Sure sounds good!